### PR TITLE
Review processing functionality

### PR DESCRIPTION
**Pull request by minnerthecat**
Added functions to parse all and single movie reviews.
Parsing removes header and footer (cast) sections from the review, converts to lowercase, removes punctuation, and tokenizes the full review text.

### CODE REVIEW

**Comment by minnerthecat** [comment]
Merging this branch requires an update to your DB schema. You'll need to add an a column called 'itemset' of type varchar.

File: get_db_with_reviews.sh [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#!/bin/bash
-
-wget https://www.dropbox.com/s/gly3og1tzz2dni5/movies.db?dl=0 -O ./movies.db
```
Comment (MattMcMurray) [inline get_db_with_reviews.sh:3]: Any idea why this is deleted? Looks like a weird git error to me, potentially.

**Review by MattMcMurray — CHANGES_REQUESTED** [review]
Everything looks good. If you can just re-add the get_db_with_reviews.sh/figure out why it was deleted, you can merge everything in after.

**Comment by minnerthecat** [comment]
That's on me, I hadn't noticed it before and when I did I thought it was something I had added by accident.

**Review by MattMcMurray — APPROVED** [review]

**Pull request merged**